### PR TITLE
fix(liveness): add deprecated comment to unused strings

### PR DESCRIPTION
**Pull request by thaddmt**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export type HintDisplayText = {
   hintCanNotIdentifyText?: string;
   hintTooCloseText?: string;
   hintTooFarText?: string;
-  hintHoldFacePositionCountdownText?: string;
+  hintHoldFacePositionCountdownText?: string /** @deprecated String is no longer in use */;
```
Comment (reesscot) [inline packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts:8]: does this show in intellisense?

Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts]: Fixed this and now it shows correctly in intellisence

**Review by reesscot — DISMISSED** [review]
LGTM, thanks for confirming

File: packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ export type HintDisplayText = {
   hintCanNotIdentifyText?: string;
   hintTooCloseText?: string;
   hintTooFarText?: string;
+  /** @deprecated String is no longer in use */
```
Comment (calebpollman) [inline packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts:8]: What do you think of adding some communication regarding removal here?
```suggestion
 /** @deprecated `hintHoldFacePositionCountdownText` is no longer in use and will be removed in a future major version release. */
```

File: packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts [hunk 3]
```diff
@@ -109,6 +110,7 @@ export const defaultLivenessDisplayText: Required<LivenessDisplayText> = {
   hintCanNotIdentifyText: 'Move face in front of camera',
   hintTooCloseText: 'Move back',
   hintTooFarText: 'Move closer',
+  /** @deprecated String is no longer in use */
```
Comment (calebpollman) [inline packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts:113]: Same question as above
```suggestion
 /** @deprecated `hintHoldFacePositionCountdownText` is no longer in use and will be removed in a future major version release. */
```

File: packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts (hunk 2 above)
Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts:8]: I'm on board with it

**Review by reesscot — APPROVED** [review]

**Review by calebpollman — APPROVED** [review]
🚢

**Pull request merged**